csharp/IceGrid/Greeter/application-with-replication.xml

```diff
@@ -0,0 +1,29 @@
+<!-- Copyright (c) ZeroC, Inc. All rights reserved. -->
+
```

Comment:
Would be nice to have comments explaining the deployment in this file.

Reply:
Yes, will do in a follow-up PR.